### PR TITLE
feat(profile): implement new picture style

### PR DESCRIPTION
**Pull request by satish-ravi**
### Description

Part 2 of ACT-662. 

https://user-images.githubusercontent.com/5062591/226709989-66929e53-8158-4861-8ad4-354ef21a4cf5.mp4

### Test plan

Unit tests, manual

### Related issues

- Fixes ACT-662

### Backwards compatibility

Yes

### CODE REVIEW

File: src/onboarding/registration/PictureInput.tsx [hunk 1]
```diff
@@ -108,34 +112,24 @@ function PictureInput({ picture, onPhotoChosen, backgroundColor }: Props) {
...
-    height: 32,
+  editIconContainer: {
+    width: 40,
+    height: 40,
     position: 'absolute',
-    right: 0,
-    bottom: 0,
-    borderRadius: 16,
-    backgroundColor: colors.greenUI,
+    left: 37,
```
Comment (jophish) [inline src/onboarding/registration/PictureInput.tsx:126]: should this be `PICTURE_SIZE / 2`? or is it unrelated?

**Review by jophish — APPROVED** [review]
LGTM, just one super tiny comment

File: src/onboarding/registration/PictureInput.tsx (hunk 1 above)
Comment (satish-ravi) [reply src/onboarding/registration/PictureInput.tsx:126]: It's unrelated

Comment (jophish) [reply src/onboarding/registration/PictureInput.tsx:126]: omg my basic math skills completely failed me :facepalm:

**Pull request merged**